modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/RelatedObjectEntryResourceTest.java

```diff
@@ -466,6 +466,41 @@ public void testGetSystemObjectRelatedObjects() throws Exception {
 			_OBJECT_FIELD_VALUE, itemJSONObject.getString(_OBJECT_FIELD_NAME));
 	}
 
+	@Test
+	public void testGetSystemObjectRelatedObjectsWithPagination()
+		throws Exception {
+
+		JaxRsApplicationDescriptor jaxRsApplicationDescriptor =
+			_userSystemObjectDefinitionMetadata.getJaxRsApplicationDescriptor();
+
+		_objectRelationship = _addObjectRelationship(
```

Comment:
Could we test here the one to many relationship too?

Reply:
will do on a follow up.